.github/workflows/deploy_branch_maven.reusable.workflow.yml

```diff
@@ -270,3 +294,5 @@ jobs:
 
       - name: Send email workflow status
         uses: ./.github/symphony-devops-workflows/actions/send_email_workflow_status
+        with:
+          builderSmtpHost: ${{ secrets.builderSmtpHost }}
```

Comment:
`builderSmtpHost` is marked as required in the action, but as not required in the workflow
So action would failbeing called if not present
Either it is mandatory everywhere, either if not we need to handle case properly and not call the action

Reply:
made it mandatory everywhere